tools/istio-iptables/pkg/capture/run.go

```diff
@@ -376,8 +377,8 @@ func (cfg *IptablesConfigurator) Run() {
 	// Use this chain also for redirecting inbound traffic to the common Envoy port
 	// when not using TPROXY.
 
-	cfg.iptables.AppendRule(iptableslog.InboundCapture, constants.ISTIOINREDIRECT, constants.NAT, "-p", constants.TCP, "-j", constants.REDIRECT,
-		"--to-ports", cfg.cfg.InboundCapturePort)
+	cfg.iptables.AppendRule(iptableslog.InboundCapture, constants.ISTIOINREDIRECT, constants.NAT, "-p", constants.TCP, "-j", constants.DNAT,
```

Comment:
See https://github.com/istio/istio/issues/36040#issuecomment-1045360454 for details on why this change was made